src/vs/workbench/contrib/comments/browser/commentThreadZoneWidget.ts

```diff
@@ -363,15 +366,13 @@ export class ReviewZoneWidget extends ZoneWidget implements ICommentThreadWidget
 			}
 
 			if (shouldMoveWidget && this._isExpanded) {
-				this.show({ lineNumber, column: 1 }, 2);
+				this.show(this.arrowPosition(this._commentThread.range), 2);
```

Comment:
Interesting. `lineNumber` was the _start_ line number. Seems like it was a bug.

Reply:
Good catch! I should change line 359 to use `endLineNumber`, right?

Reply:
Actually, maybe not. In the multi-line case the glyph goes on the start line and the zone widget after the end line, correct?

Reply:
I think the glyph should always go on the same line as the zone widget, so they should both be on the end line. This is fine as is, I'll go through and look at the start line/end line uses.